city_scrapers/spiders/chi_ssa_50.py

```diff
@@ -0,0 +1,77 @@
+from datetime import datetime
+
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa50Spider(CityScrapersSpider):
+    name = "chi_ssa_50"
+    agency = "Chicago Special Service Area"
+    timezone = "America/Chicago"
+    start_urls = ["http://southeastchgochamber.org/special-service-area-50/"]
+    base_xpath = '//div[@class="entry-content"]'
+    item_xpath = base_xpath + '//p[@style="text-align: center;"]/text()'
+
+    def parse(self, response):
+        title = self._parse_title(response)
+        description = self._parse_description(response)
+
+        for item in response.xpath(self.item_xpath).getall():
+            meeting = Meeting(
+                title=title,
+                description=description,
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=self._parse_end(item),
+                all_day=self._parse_all_day(),
+                time_notes=self._parse_time_notes(),
+                location=self._parse_location(),
+                links=self._parse_links(),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
```

Comment:
Just in case a meeting is ever cancelled it would be safer to pass `item` as a `kwarg` to `_get_status` to check if it's mentioned there

Reply:
Ok, I will make the change

Reply:
Something like `meeting["status"] = self._get_status(**item)`?

Reply:
In this case it would be

```python
meeting["status"] = self._get_status(meeting, text=item)
```